docs/guides/monitoring.md

```diff
@@ -86,7 +86,7 @@ groups:
 - name: GitOpsToolkit
   rules:
   - alert: ReconciliationFailure
-    expr: gotk_reconcile_condition{type="Ready",status="False"} == 1
+    expr: (gotk_reconcile_condition{status="False",type="Ready"} + on(exported_namespace, name, kind) (gotk_reconcile_condition{status="Deleted"} * 2)) == 1
```

Comment:
Have this been tested? There is no such label in our exporters `exported_namespace`

Reply:
Oh indeed, it maybe on my prometheus relabelling 👍 I'll try with a raw scraper